src/content/peerconnection/old-new-stats/js/main.js

```diff
@@ -0,0 +1,289 @@
+/*
```

Comment:
There's quite a lot of stuff here that doesn't follow Google JavaScript style and will cause linting errors: semi-colons at the end of every line, spacing around function names, etc. 

I think you should run all the files through ESLint (using [.eslintrc](https://github.com/webrtc/samples/blob/gh-pages/.eslintrc)) and fix accordingly.

Same for CSSLint if necessary.